app/components/primer/alpha/dialog.pcss

```diff
@@ -8,8 +8,15 @@
 
 /* Overlay */
 
+@property --dialog-scrollgutter {
```

Comment:
nit: can we comment here, maybe with a link to this issue?  future us might wonder why we're doing it